Tests/resources/OrderHelperData.php

```diff
@@ -0,0 +1,229 @@
+<?php
+/**
+ * Shop System Plugins:
+ * - Terms of Use can be found under:
+ * https://github.com/wirecard/oxid-ee/blob/master/_TERMS_OF_USE
+ * - License can be found under:
+ * https://github.com/wirecard/oxid-ee/blob/master/LICENSE
+ */
+
+namespace Wirecard\Oxid\Tests\resources;
+
+class OrderHelperData
+{
+    const DB_DATA = [
+        [
+            'table' => 'oxuser',
+            'columns' => [
+                'OXID',
+                'OXACTIVE',
+                'OXUSERNAME',
+                'OXFNAME',
+                'OXLNAME',
+                'OXCOMPANY',
+                'OXSTREET',
+                'OXSTREETNR',
+                'OXZIP',
+                'OXCITY',
+                'OXCOUNTRYID',
+                'OXSTATEID',
+            ],
+            'rows' => [
+                [
+                    'User ID 1',
+                    1,
+                    'User 1',
+                    'User',
```

Comment:
I would suggest making use of the TEST_USER_4 data (starting from 'User') as it makes the correlation more obvious while reducing redundancy.